app/components/webauthn_input_component.rb

```diff
@@ -36,4 +41,12 @@ def initial_hidden_tag_options
       { class: 'js' }
     end
   end
+
+  def show_desktop_ft_unlock_option?
+    if desktop_ft_unlock_option? && I18n.locale == :en
```

Comment:
I think we need to explicitly check whether the bucket is [the `:desktop_ft_unlock_option_shown` bucket](https://github.com/18F/identity-idp/blob/3db4df3bf6f5b2461985e92e2183b7741c505ddd/config/initializers/ab_tests.rb#L112) somewhere. Otherwise, I'd be concerned that [the default bucket `:default`](https://github.com/18F/identity-idp/blob/9f9aaf8f78252be208b14d003b5f24a6652d644c/lib/ab_test.rb#L22) is "present", and we'll be showing the option for 100% of desktop users.

This might be easy to miss in development since we're configuring the A/B test at 100% anyways.